gradle/check-licenses.gradle

```diff
@@ -168,6 +168,7 @@ downloadLicenses {
 
     // jnr-posix is released under a tri EPL v2.0/GPL/LGPL license
     'com.github.jnr:jnr-posix:3.0.47' : epl2,
+    'com.github.jnr:jnr-posix:3.1.15' : epl2,
```

Comment:
I tried grouping these and omitting the version numbers but the licence checker fails - seems to find GPL first https://app.circleci.com/pipelines/github/hyperledger/besu/14696/workflows/0f4cedc3-a8e8-4c42-90bd-019841000303/jobs/84335

Reply:
Also - even though the constraint requires at least 3.1.15, the licence checker still picks up 3.0.47 - but it does not appear in the dependency tree according to `gradle -q dependencies`